### PR TITLE
Update Project Dependencies

### PR DESCRIPTION
**Pull request by jason-klein**
We need to update project dependencies on a regular basis. I updated dependencies and fixed problems that were introduced after the update. I also added a few pending items to the README file.

### CODE REVIEW

**Comment by jason-klein** [comment]
@jugglingdev  Ready for review and merge

File: README.md [hunk 1]
```diff
@@ -9,13 +16,23 @@
...
 * For running sail commands, by default, you are required to enter the full path to the executable in `vendor/bin/sail`. Most devs prefer to create an alias in their shell so they only have to type `sail`. Read the sail docs about [configuring a sail alias](https://laravel.com/docs/11.x/sail#configuring-a-shell-alias). Further documentation will assume an alias exists in your shell.
 
-## Updating the Project
-* If you get an error with the `composer.lock` file, you may need to update your project
-* To update, navigate to the project directory and run `composer update`
+## Installing Current Project Dependencies
+* All project contributors should run these commands every week to ensure your local project is using the current project dependencies.
+  * `composer install`
+  * `npm install`
+  * `npm run build`
```
Comment (jugglingdev) [inline README.md:23]: Zach was hitting an error with this that he posted in Good Dads.  It might be good to include that in a Troubleshooting section.

Comment (Xacheri) [reply README.md:23]: I ran a git reset --hard and reinstalled all packages. It worked, all clear,

**Review by jugglingdev — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jugglingdev) [reply README.md:23]: Thank you for the update, Zach!

**Comment by jugglingdev** [comment]
I went ahead and approved the changes here.  If we'd like to add a Troubleshooting section in the future, I think it would be good to do.  Thank you, Jason!

**Pull request merged**